ui/src/components/GlobalSearch.svelte

```diff
@@ -220,6 +220,7 @@
           class="sm:hidden text-tertiary"
           on:click={() => {
             searchInput.blur();
+            setIsSearching(false);
```

Comment:
I expect .blur to do this already, but for some reason clicking these buttons on mobile didn't hide search results. maybe it's not a "click"